README.md

```diff
@@ -320,6 +321,11 @@ spec:
 
 For the full list of arguments available, see the documentation in [docs/cli-arguments.md](./docs/cli-arguments.md)
 
+
+#### Helm Chart
+
+A helm chart is maintained in [prometheus-community/helm-charts](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics).
```

Comment:
Should we state that it is if the `official helm chart` for `kube-state-metrics`?

Reply:
good idea! I updated it.